### PR TITLE
Product landing page

### PR DESCRIPTION
**Pull request by Hasinarahman**
Capture
clone the repo in local machin.

### CODE REVIEW

**Comment by Hasinarahman** [comment]
hello teacher @ahzamir
I done the product landing page project please check it and give me feedback.

File: README.md
Comment (ahzamir) [inline README.md]: - Remove this part

File: README.md
Comment (ahzamir) [inline README.md]: - Add your project as a demo (Screenshot)

File: README.md
Comment (ahzamir) [inline README.md]: - You need to add your LinkedIn account here

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although you have done a very great job by implementing some of the requirements, but still you need to work on your README.md file in order to get approval for your project
  - Check the inline comments below
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.

image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Product Landing Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
❌You have a professional README.md file in your project.

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hasinarahman** [comment]
hello sir @ahzamir 
I update the readme file please check it.

**Comment by Hasinarahman** [comment]
@ahzamir 
Hi sir I am waiting for your feedback.

**Review by ahzamir — APPROVED** [review]
### Hi @Hasinarahman 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Add some more design to your project so you can share it.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Product Landing Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Hasinarahman** [comment]
ok Thanks you sir
@ahzamir

**Pull request merged**
